packages/client/components/MeetingsDash.tsx

```diff
@@ -113,7 +122,7 @@ const MeetingsDash = (props: Props) => {
           <MeetingsDashEmpty
             name={preferredName}
             message={
-              dashSearch
+              dashSearch || teamFilter
                 ? EmptyMeetingViewMessage.NO_SEARCH_RESULTS
```

Comment:
+1 I don't think the existing `NO_SEARCH_RESULTS` copy is effective when the user is only filtering by team, since it specifies "query" (as in, search query), and it may not be clear that you're filtering down to a specific team.

IMO ideally, this would read something like `we could not find any meetings on this team`, with a link that looks like `See meetings on all teams` immediately after.